src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/NewAzureADApplicationCommand.cs

```diff
@@ -39,19 +39,6 @@ public class NewAzureADApplicationCommand : ActiveDirectoryBaseCmdlet
         public string DisplayName { get; set; }
 
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ApplicationWithoutCredential,
-            HelpMessage = "The URL to the application’s homepage.")]
```

Comment:
As above, add SuppotsShouldProcess=true to the cmdlet attribute, and protect execution with a call to ShouldProcess